ci/container/wss.sh

```diff
@@ -5,25 +5,33 @@ set -e
 set -o pipefail
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-
 [[ -z "$WHITESOURCE_API_KEY" ]] && echo "[WARNING] No WHITESOURCE_API_KEY is set. No WhiteSource scan will occur." && exit 0
 
 export PRODUCT_NAME=snowflake-jdbc
-export PROJECT_NAME=snowflake-jdbc
 
-DATE=$(date +'%m-%d-%Y')
+export PROD_BRANCH=master
+export PROJECT_VERSION=$GITHUB_SHA
+
+if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
+    echo "[INFO] Pull Request"
+    IFS="/"
+    read -ra GITHUB_REF_ELEMS <<<"$GITHUB_REF"
+    IFS=" "
+    export PROJECT_NAME=PR-${GITHUB_REF_ELEMS[2]}
+elif [[ "$GITHUB_HEAD_REF" == "$PROD_BRANCH" ]]; then
+    echo "[INFO] Production branch"
+    export PROJECT_NAME=$GITHUB_HEAD_REF
+else
+    echo "[INFO] Non Production branch. Skipping wss..."
+    export PROJECT_NAME=
+fi
 
 SCAN_DIRECTORIES=$(cd $THIS_DIR/../.. && pwd)
 
-rm -f wss-unified-agent.jar
-curl -LO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
-
-# whitesource will scan the folder and detect the corresponding configuration
-# configuration file wss-generated-file.config will be generated under ${SCAN_DIRECTORIES}
-# java -jar wss-unified-agent.jar -detect -d ${SCAN_DIRECTORIES}
-# SCAN_CONFIG="${SCAN_DIRECTORIES}/wss-generated-file.config"
-
-# SCAN_CONFIG is the path to your whitesource configuration file
+if [[ -n "$PROJECT_NAME" ]]; then
+    rm -f wss-unified-agent.jar
```

Comment:
hello, @smtakeda may I know why remove/download wss agent jar file is under this if condition? is there anything related with project name here?